index.html

```diff
@@ -41,8 +41,167 @@ <h1 class="primary">
                </ul>
             </div>
             
-            <img src="assets/bottom arrow.svg" alt="bottom-arrow">
         </section>
+        
+        <div class="bottom-arrow">
+            <img src="assets/bottom arrow.svg" alt="bottom-arrow" >
+        </div>
+        
+        <section class="heading">
+            <h1 class="color1 primary">My Recent <br>Works</h1>
+            <img src="assets/indicator.svg" class="indicator" alt="indicator">
+            
+            <div class="project-grid">
+                <div class="project-container">
+                    <div class="img-container"></div>
+                    <div class="project-details">
+                        <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
+                    
```

Comment:
Kindly, I suggest you remove all unnecessary white lines in your code base to make your project look more professional and for best practices. 😄